master/static/srv/get_experiments.sql

```diff
@@ -1,15 +1,33 @@
+WITH filtered_exps AS (
```

Comment:
For posterity on why we went this way: yes this CTE is going to be fairly expensive, but still less expensive in some aspects than what we're doing right now (maybe less network/same disk io/more cpu on the db?) and the master will be _much_ better off for it, too.

Reply:
I don't know a way to make the SELECT operations themselves cheaper unless we add indexes to fasten executing `WHERE`.

Reply:
I just mean that I dont think postgres 10.x (or later tbh) can do anything special with this and the CTE will be fully materialized just for the count. I don't think their is anyway around this given the API constraints from the webui.

Reply:
Right. Unless we get rid of the total count of the filtered experiments from the API, there is no way to work around this.